tsconfig.json

```diff
@@ -8,7 +8,12 @@
     "inlineSourceMap": true,
     "declaration": true,
     "newLine": "LF",
-    "outDir": "dist"
+    "outDir": "dist",
+    "lib": [
```

Comment:
This shouldn't be necessary since we should automatically get the es2017 lib https://github.com/Microsoft/TypeScript/blob/6117ed77087b5d84774ee545bd3cad3d059caa75/src/compiler/utilities.ts#L4245